### ISSUE TITLE
Several command param lengths are incorrect

### ISSUE THREAD

**Issue by inexorabletash** [inexorabletash/a2d#60]
* [x] `SetCtlMax`: documented as taking 2 bytes, table gives it 3
* [x] `UpdateThumb`: documented as taking 2 bytes, table gives it 3
* [x] `KeyboardMouse`: documented as taking no parameters, but does copy/ignore one byte
* [x]  And the markdown docs for `TrackGoAway` confusingly indicate extra params.

**Comment by eric-ja** [comment]
>  TrackGoAway: documented as returning 1, appears to return 3

I don't see how you got 3? There's only one `sta (params_addr),y`.

**Comment by eric-ja** [comment]
>  SetCtlMax: documented as taking 2 bytes, table gives it 3
UpdateThumb: documented as taking 2 bytes, table gives it 3

Yeah, the table is wrong (correct length is 2). But copying an extra byte from params should be harmless.

**Comment by inexorabletash** [comment]
Sorry, this was based on notes I jotted down but didn't turn into a bug. I didn't review the code again, it was probably from much earlier in the deciphering. 

I have the nagging feeling I did find one call that was trashing memory after the documented params, but that may have been before I understood the in/our convention.

(As you've noticed there are incorrect comments in a few places in MGTK)
